agent/structs/check_definition.go

```diff
@@ -54,6 +54,17 @@ func (c *CheckDefinition) HealthCheck(node string) *HealthCheck {
 	if health.CheckID == "" && health.Name != "" {
 		health.CheckID = types.CheckID(health.Name)
 	}
+
```

Comment:
After talking with @banks maybe we should wait on this part until we can determine if we should include the data. We can always PR this later.